spec/rmagick/class_methods/_tmpnam__spec.rb

```diff
@@ -0,0 +1,50 @@
+module Magick
+  def self._tmpnam_
+    @@_tmpnam_
+  end
+end
+
+RSpec.describe Magick, '#tmpnam' do
```

Comment:
Do we not need the underscores here? (not a ruby expert so this might be a dumb question)

Reply:
Oh, yeah, those should be there. It's just a string description, not a ruby syntax issue, so it should match the file/method name.

Reply:
@dlemstra updated. I also changed the `#` to a `.`, since `#` indicates instance methods and `.` is for class methods.